.github/workflows/test-template.yml

```
@@ -961,7 +961,7 @@ jobs:
   # minimize the time spent waiting for a free runner.
   #
   no-docker-client-functional:
-    name: Client (functional tests running on Windows/macOS, no-docker)
+    name: Client (func Win/mac)
```

Isn't the "no docker" important info?
```suggestion
    name: Client (func Win/mac no-docker)
```

---

it's too long, also not that "important", there is no need to have both win/mac and no-docker in the same name, it's equivalent for this job